src/timeline/InputBar.cpp

```diff
@@ -251,12 +251,14 @@ InputBar::openFileSelection()
 }
 
 void
-InputBar::message(QString msg)
+InputBar::message(QString msg, MarkdownOverride useMarkdown)
 {
         mtx::events::msg::Text text = {};
         text.body                   = msg.trimmed().toStdString();
 
-        if (ChatPage::instance()->userSettings()->markdown()) {
+        if ((ChatPage::instance()->userSettings()->markdown() &&
+             (useMarkdown != MarkdownOverride::OFF)) ||
+            (useMarkdown == MarkdownOverride::ON)) {
```

Comment:
Shouldn't one of them be enough?

Reply:
`(ChatPage::instance()->userSettings()->markdown() && (useMarkdown != MarkdownOverride::OFF))` makes sure that MD is on and that "/plain" wasn't specified. `(useMarkdown == MarkdownOverride::ON)` is supposed to check if "/md" was specified. I don't think there's any major simplifications that I can do here.